d4rl/setup.py

```diff
@@ -8,7 +8,7 @@
     install_requires=[
         "gym",
         "numpy",
-        "mujoco_py",
+        "mujoco-py",
```

Comment:
Why is this necessary? The install runs fine without this. (same for all replacements of mujoco_py with mujoco-py.